packages/eslint-config-typescript/rules/typescript-eslint.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  overrides: [
+    {
+      files: ['**/*.ts?(x)'],
+      rules: {
+        '@typescript-eslint/no-unused-vars': [
+          'error',
+          {
+            argsIgnorePattern: '^_',
```

Comment:
Can we add a note about what this is doing?

Reply:
Added in https://github.com/reside-eng/lint-config/pull/343/commits/8c5b914a5d6c5bfdcb9e7ac5dc81c4edb29031e4!